.github/workflows/ci.yml

```diff
@@ -175,16 +175,23 @@ jobs:
         with:
           name: "unit tests"
   smoke-tests:
-    name: Smoke tests ${{ matrix.os }} on ${{ matrix.arch }}
+    name: Smoke tests on ${{ matrix.os }} with ${{ matrix.arch }}
     timeout-minutes: 20
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.runner }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ macos-latest, windows-latest, ubuntu-latest ]
+        os: [ macos, windows, linux ]
         arch: [ amd64 ]
         include:
-          - os: ubuntu-latest
+          - os: macos
+            runner: macos-latest
+          - os: windows
+            runner: windows-latest
+          - os: linux
+            runner: n1-standard-8-netssd-preempt
+          - os: linux
+            runner: n1-standard-8-netssd-preempt
```

Comment:
curious to try those once more, will alternatively ping infra on a `quick` pool for 8 core runners